spec/cc/analyzer/issue_spec.rb

```diff
@@ -16,72 +16,44 @@ module CC::Analyzer
         },
         "remediation_points" => 10,
         "type" => "issue",
-      }.to_json
-      issue = Issue.new(output)
+      }
+    end
+
+    it "allows access to keys as methods" do
+      issue = Issue.new(output.to_json)
 
       expect(issue.respond_to?("check_name")).to eq true
       expect(issue.check_name).to eq("Rubocop/Style/Documentation")
     end
 
     describe "#fingerprint" do
+      include EnvHelpers
```

Comment:
I thought I converted this test suite to RSpec? I think we typically inject our helper modules into all specs via `RSpec.configure` immediately after the module definition, and only do this sort of thing when we're forced to by minitest.